src/StochasticStyles/spawning.jl

```diff
@@ -12,8 +12,8 @@ end
 # Non-integer
 @inline function projected_deposit!(
     ::Type{T}, w, add, value, parent, threshold
-) where {T<:Real}
-    thresh = T(threshold)
+) where {T<:Union{AbstractFloat, Complex{<:AbstractFloat}}}
+    thresh = abs(T(threshold))
```

Comment:
Cool to see that's enough to make it work!

Reply:
Yes, very neat!

I'd like to see some test for this functionality for existing projector Monte Carlo with complex populations and complex Hamiltonians (e.g. complex hopping parameters in Hubbard models that keep the Hamiltonian Hermitian). It also should be separated from the real-time evolution stuff into a separate PR.